services/repository/pom.xml

```diff
@@ -249,13 +249,13 @@
 					<destDir>${project.build.directory}/javadoc/restdocs</destDir>
 					<!-- This name is used to link to the authentication controller -->
 					<additionalJOption>-authControllerName org.sagebionetworks.auth.AuthenticationController</additionalJOption>
-					<additionnalDependencies>
```

Comment:
this is maven's typo
https://maven.apache.org/components/plugins-archives/maven-javadoc-plugin-2.9.1/examples/additionnal-dependencies.html

Reply:
as a note this is fixed in later versions but upgrading the version breaks the document generation. not sure what causes it, but fixing the typos is more work than a simple version bump